pkg/sql/unsafe/unsafe.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2025 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+package unsafe
```

Comment:
I think we should name this package `unsafesql` or something else that avoids ambiguity with the builtin go `unsafe` package. Seeing something like `unsafe.Foo()` in some code typically sets off alarm bells, but it shouldn't here (the opposite in fact) but you would need to see which "unsafe" was imported to know that.

Reply:
Ah that's a great point, yeah I'll go with `unsafesql`